app/controllers/registrations_controller.rb

```diff
@@ -666,16 +735,18 @@ def refund_payment
       return redirect_to edit_registration_path(registration)
     end
 
-    # Backwards compatibility: We may at some point try to record a refund for a payment that was
-    #   - created before the introduction of receipts
-    #   - but refunded after the new receipts feature was introduced. Fall back to the old stripe_charge_id if that happens.
-    charge_id = payment.receipt&.stripe_id || payment.stripe_charge_id
+    if payment.receipt.nil?
```

Comment:
Is this even a case we need to handle? (ie, is it just bloating our controller?)